test/testcafe/spec/SelectFactorForEnroll_spec.js

```diff
@@ -0,0 +1,25 @@
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import SelectFactorPageObject from '../framework/page-objects/SelectFactorPageObject';
+import { RequestMock } from 'testcafe';
+import factorEnrollOptions from '../../../playground/mocks/idp/idx/data/factor-enroll-options';
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx')
```

Comment:
we shall also mock `/instrospect` endpoint. It's anti pattern to leverage mocks setting from playground.
same as other tests.